includes/Core/Authentication/Authentication.php

```diff
@@ -1202,29 +1207,23 @@ private function get_proxy_permissions_url() {
 		);
 	}
 
-
 	/**
 	 * Verifies the user input settings
 	 *
 	 * @since 1.20.0
 	 */
 	private function verify_user_input_settings() {
 		if (
-			! empty( $this->user_input_state->get() )
-			|| ! $this->is_authenticated()
-			|| ! $this->credentials()->has()
-			|| ! $this->credentials->using_proxy()
+			empty( $this->user_input_state->get() )
+			&& $this->is_authenticated()
+			&& $this->credentials()->has()
+			&& $this->credentials->using_proxy()
 		) {
-			return;
-		}
```

Comment:
Nit-picking, but why turn this condition around? I think where possible we should use early returns to avoid deep nesting. Of course, the nesting here is only one level deep, but I think sticking to this pattern overall makes sense and avoids unnecessary code changes when later indenting differently, making code review unnecessarily harder.

Reply:
I thought it would be easier to read if I combined it. Do you want me to revert it?

Reply:
Thanks, but not a big deal. Just overall I think we should prefer early returns over nesting where possible.